Assets/_BForBoss/_Utility/Scripts/ObjectPooler.cs

```diff
@@ -48,7 +48,7 @@ public ObjectPooler(string poolName, Func<T> buildObject, Action<T> actionOnGet
 		~ObjectPooler()
         {
 #if UNITY_2021_1_OR_NEWER
-			objectPool.Dispose();
+			objectPool.Clear();
 #else
 			SceneManager.UnloadSceneAsync(poolScene); //destroys all objects in the pool
```

Comment:
Shouldn't the rest of the code be removed?

Reply:
Sure I'll do that